src/GDriveCloudStorage.js

```diff
@@ -0,0 +1,50 @@
+import GDrive from 'react-native-google-drive-api-wrapper';
+import { GoogleSignin } from '@react-native-community/google-signin';
+
+GoogleSignin.configure({
+  scopes: [
+    'https://www.googleapis.com/auth/drive.file',
+    'https://www.googleapis.com/auth/drive.appdata',
+    'https://www.googleapis.com/auth/drive.metadata',
+  ],
+});
+
+const { accessToken } = GoogleSignin.getTokens();
+
+GDrive.setAccessToken(accessToken);
+GDrive.init();
+
+if (!GDrive.isInitialized) {
+  throw new Error('Unable to use G drive');
+}
```

Comment:
Wrap this code in a `login()` function. Otherwise this would run automatically in the global scope when the module is loaded in the unit tests.

Reply:
Yeh good shout, given it a slightly different name as this code will assume the user is already logged in. I don't think we want to handle any login flows or anything here.